R/pkg/R/SQLContext.R

```diff
@@ -62,21 +62,23 @@ infer_type <- function(x) {
   }
 }
 
-#' Create a DataFrame from an RDD
+#' Create a DataFrame
 #'
-#' Converts an RDD to a DataFrame by infer the types.
+#' Converts R data.frame or list into DataFrame.
```

Comment:
I just thought that the logic for supporting RDD is here and users may use SparkR::: to access RDD functions. I am neutral to whether we put RDD in this description or not. @shivaram, what's your comment?